packages/tray/test/tray.test.ts

```diff
@@ -100,4 +104,162 @@ describe('Tray', () => {
 
         expect(el.open).to.be.false;
     });
+
+    describe('Dismiss helpers', () => {
+        it('renders visually-hidden dismiss helpers when no buttons detected', async () => {
```

Comment:
We should also text that this button can be accessed via keyboard to dismiss the tray.